auth/AuthSample/Program.cs

```diff
@@ -335,12 +335,12 @@ public class CloudLibrary : AuthLibrary
         // method.
         public object AuthExplicit(string projectId, string jsonPath)
         {
-            var credential = GoogleCredential.FromFile(jsonPath)
-                .CreateScoped(LanguageServiceClient.DefaultScopes);
-            var channel = new Grpc.Core.Channel(
-                LanguageServiceClient.DefaultEndpoint.ToString(),
-                credential.ToChannelCredentials());
-            var client = LanguageServiceClient.Create(channel);
+            LanguageServiceClientBuilder builder = new LanguageServiceClientBuilder

```

Comment:
I'd probably do all of this in a single line:

```csharp
LanguageServiceClient client = new LanguageServiceClientBuilder { CredentialsPath = jsonPath }.Build();
```

That's more the kind of code I expect our users to write.